go/gildedrose/updatestrategy/defaultUpdateStrategy.go

```diff
@@ -0,0 +1,32 @@
+package updatestrategy
```

Comment:
this file contains the default update strategy for both sellin and quality

by defining the below code, we are embedding the separate sellin and quality strategies inside the default overall strategy
```type defaultUpdateStrategy struct {
	defaultSellinUpdateStrategy
	defaultQualityUpdateStrategy
}```